src/app/views/home/Home.js

```diff
@@ -158,11 +173,8 @@ export default class Home extends React.Component {
                   <img id="venushacks-title" src={vh_title} alt="VenusHacks Title Logo"/>
                   <h4 id="date">May 20 - 22, 2022</h4>
                   <p id="tagline">UC Irvine's largest women-centric hackathon</p>
-                    <Button id="apply-btn">
-                      <a href="https://airtable.com/shr1Etr4Typ3FEqBc" target="_blank" rel="noopener noreferrer">
-                        Apply as a Hacker
-                      </a>
-                    </Button>
+                      {this.ApplyButton("https://airtable.com/shr1Etr4Typ3FEqBc", "Apply as a Hacker")}
+                      {this.ApplyButton("https://airtable.com/shr5Y9x5ocJaJL52j", "Apply as a Mentor")}
```

Comment:
When we import it as a component, we no longer need `this` and we can be explicit with the props.
```suggestion
                      <ApplyButton url="https://airtable.com/shr1Etr4Typ3FEqBc" text="Apply as a Hacker" />
                      <ApplyButton url="https://airtable.com/shr5Y9x5ocJaJL52j" text="Apply as a Mentor" />
```